sensor/common/delegatedregistry/delegated_registry_handler.go

```diff
@@ -0,0 +1,160 @@
+package delegatedregistry
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/centralsensor"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/images/utils"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/sensor/common"
+	"github.com/stackrox/rox/sensor/common/registry"
+	"github.com/stackrox/rox/sensor/common/scan"
+	"google.golang.org/grpc"
+)
+
+var (
+	log = logging.LoggerForModule()
+
+	scanTimeout         = 6 * time.Minute
```

Comment:
Probably should make this one configurable via environment variable, since it may vary based on the size of image we are scanning etc. and currently there'd be no way to help any customers that run into timeouts.

Reply:
I copied the value from [here](https://github.com/stackrox/stackrox/blob/f1abdaf371b14126c746530d96a2d634eec4401e/sensor/common/detector/enricher.go#L30) which was also not configurable in Sensor - I'm not opposed to making both an env var. Will make the update